src/config/app.ts

```diff
@@ -249,6 +249,12 @@ export const getBuildVersions = (): {
 
 export const PROXY_CHECK_APIKEY = yamlConfig?.PROXY_CHECK_APIKEY
 
+// FIXME: we have process.env.NODE_ENV === "production" | "development" | "test"
+// "test" might not be needed
```

Comment:
Why is this comment necessary? Where are we setting or depending on `NODE_ENV="test"`?

Reply:
don't know. the comment with `FIXME` is that we look into it in future PR.